pkgs/development/libraries/qt-5/5.14/qtbase.patch.d/813a928c7c3cf98670b6043149880ed5c955efb9.patch

```diff
@@ -0,0 +1,53 @@
+Non-conflicting bits from
```

Comment:
I think you commited this file by accident.

Reply:
Great catch! Yeah, I was picking needed hunks from that file and committed it by accident. Removed.